components/site/src/sass/mod.rs

```diff
@@ -52,6 +55,24 @@ pub fn compile_sass(base_path: &Path, output_path: &Path) -> Result<()> {
     Ok(())
 }
 
+/// write out a subset of the Zola config document to a temporary SCSS file
+/// as an SCSS map variable literal.  this will allow parts of the site's
+/// config to be usable during Sass compilation.  this enables theme configuration
+/// like allowing the site owner to change header color.  this function returns
+/// a tempdir holding a single `.scss` file.  the tempdir should then be used as
+/// a load directory above when compiling the site's Sass files.  the tempdir
+/// and contained `.scss` file will be deleted on drop of the returned `TempDir`
+/// struct, which should happen after Sass compilation finishes.
+fn build_dependencies_dir_from_config(config: &Config) -> Result<TempDir> {
```

Comment:
I don't understand the name of that function.

Reply:
here we take the config document and construct a directory that contains files that are ultimately added as dependencies when compiling Sass.  the Sass compiler asks for a "load dir" during compilation, so instead of providing a single file we must provide a single directory containing a single file.  so the function name is basically "builds a dependencies directory from the config document".  happy to rename, do you have a suggestion?

Reply:
`write_temp_config_file`?

Reply:
that works for me.  seems odd to me that a function would be named that but return a directory instead of a file, but whatever.

Reply:
ah fair enough `create_config_load_path`?

Reply:
i like it